lib/features/keyboard/DiagramKeyBindings.js

```diff
@@ -0,0 +1,134 @@
+import {
+  isCmd,
+  isShift,
+  MoveCanvasFactory
+} from './helpers';
+
+var zKeys = [ 'Z', 'z' ];
+
+var copyKeys = [ 'C', 'c' ];
+var pasteKeys = [ 'V', 'v' ];
+
+var undoKeys = [ 'Z', 'z' ];
+var redoKeys = zKeys.concat('Y', 'y');
+
+var removeSelectionKeys = [ 'Delete' ];
+
+var zoomDefaultKeys = [ '0' ];
+var zoomInKeys = [ '+', 'Add', '=' ];
+var zoomOutKeys = [ '-', 'Subtract' ];
+
+var moveCanvasLeftKeys = [ 'ArrowLeft', 'Left' ];
+var moveCanvasUpKeys = [ 'ArrowUp', 'Up' ];
+var moveCanvasRightKeys = [ 'ArrowRight', 'Right' ];
+var moveCanvasDownKeys = [ 'ArrowDown', 'Down' ];
+
+var DEFAULT_PRIORITY = 1000;
+
+/**
+ * Adds default KeyboardEvent listeners
+ *
+ * @param {Keyboard} keyboard
+ * @param {EditorActions} editorActions
+ */
+export default function DiagramKeyBindings(keyboard, editorActions) {
+
+  var moveCanvasLeft = MoveCanvasFactory('left', keyboard._config, editorActions);
+  var moveCanvasUp = MoveCanvasFactory('up', keyboard._config, editorActions);
+  var moveCanvasRight = MoveCanvasFactory('right', keyboard._config, editorActions);
+  var moveCanvasDown = MoveCanvasFactory('down', keyboard._config, editorActions);
+
+  keyboard.addListener(copyKeys, DEFAULT_PRIORITY, copy);
```

Comment:
Could we support something event more powerful for our users here:

```
keyboard.addCombo([ `ctrl+A`, `ArrowUp` ], doSomething)
```

This would be a true simplification for the API.

Inspired by [hotshot](https://github.com/Shopify/hotshot).

Reply:
Thank you, I will have a look at that.